spec/checkPullRequestReviewSpec.js

```diff
@@ -1,11 +1,11 @@
-// Copyright 2020 The Oppia Authors. All Rights Reserved.
-//
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
```

Comment:
This file was authored in 2020. I don't think this should be changed.

Reply:
Will change it.